Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/common.js

```diff
@@ -197,4 +197,21 @@ function elapsedTime(startTimestamp, endTimestamp)
     return result;
 }
 
-export {deepCompare, ErrorDisplay, queryToParams, paramsToQuery, QueryModifier, escapeHTML, linkify, percentage, elapsedTime};
+function toGithubLink(file, suite) {
+    let baseURL = 'https://github.com/WebKit/WebKit/blob/main/';
+    switch (suite) {
+        case 'api-tests':
+            return '#';
+        case 'javascriptcore-tests':
+            return '#';
```

Comment:
I couldn't find a reliable way to map the API and JSC test names to their respective files, if there is a better way please let me know.